common/constants.go

```diff
@@ -155,6 +155,8 @@ const (
 	ConfigCICDTektonGitRepoBasicAuthSecretNameKey = ConfigCICDTektonKey + d + "gitrepobasicauthsecret"
 	// ConfigCICDTektonRegistryPushSecretNameKey is for Tekton push image to registry credentials
 	ConfigCICDTektonRegistryPushSecretNameKey = ConfigCICDTektonKey + d + "registrypushsecret"
+	//ConfigCICDArgoDestinationNameSpace gives the user input namespace destination
+	ConfigCICDArgoDestinationNameSpace = "destnamespace"
```

Comment:
Can you change the variable name from `ConfigCICDArgoDestinationNameSpace` to `ConfigCICDArgoCDDestinationNameSpaceKey`?

Also, can you add a new constant`ConfigCICDArgoCDKey = ConfigCICDKey + d + "argocd"` (like we have for Tekton [here](https://github.com/konveyor/move2kube/blob/18bb00d8c08885e62c24c33a704f506d7f51f771/common/constants.go#L151)), and change the value of `ConfigCICDArgoCDDestinationNameSpaceKey` to `ConfigCICDArgoCDKey + d + "destnamespace"`